From cc17d59237b152db02134afde8ffc2a39e2ba16b Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Mon, 7 Dec 2020 11:45:04 +0800
Subject: [PATCH] ARM: rockchip_defconfig: enable CONFIG_HID_NINTENDO by
 default

Test:
    android.hardware.input.cts.tests.NintendoSwitchProTest#testAllKeys
    android.hardware.input.cts.tests.NintendoSwitchProTest#testAllMotions

This config will enable the Nintendo Switch Pro controller driver.

Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
Change-Id: I4a29d458c3a0d82dec33e4b3990888217d580298
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 4ae09672aa5a..dea6528df85d 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -538,6 +538,7 @@ CONFIG_SND_SOC_SPDIF=y
 CONFIG_SND_SIMPLE_CARD=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=y
+CONFIG_HID_NINTENDO=y
 CONFIG_HID_SONY=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
-- 
2.35.3

